src/Game/MouseHandler.java

```diff
@@ -41,14 +41,17 @@
   private float m_rotateX = 0.0f;
 
   public static boolean inBounds(Rectangle bounds) {
+    if (XPBar.hoveringOverMenu || XPBar.hoveringOverBar()) {
+      return true;
```

Comment:
should probably set `XPBar.hoveringOverMenu = false;` here in case it gets stuck on & user is unable to trigger the code that re-evaluates it